definitions/actions.lua

```diff
@@ -424,7 +424,8 @@ return {
       SnapshotsSaveCurrent = "_SWSSNAPSHOT_SAVE",
       SplitAndSelectItemsInRegion = "_S&M_SPLIT11",
       SplitAtTimeSelection = 40061,
-      SplitItemsAtEditCursor = {40289, {"_XENAKIOS_SELITEMSUNDEDCURSELTX"},40757 ,40289},
+      SplitItemsAtEditCursor = {"UnselectItems", "SelectItemsUnderEditCursor", "SplitItemsUnderEditCursor","UnselectItems"},
```

Comment:
Thanks! But to be meticulous... you're missing a space `...r","Uns...`

Reply:
Just do a quick commit to fix that and i'll merge this right away

Reply:
Gah! Will do